### DISCUSSION TITLE
Rofi exception transparency

### DISCUSSION THREAD

**Discussion by josianjehu** [yshui/picom#740] [Q&A]
Cant get the `"100:class_g ='Rofi'"` rule to work any suggestions? I am running the following command for my popup menus `rofi -show drun`. All of my other rules are working as expected.

**Comment by vincentbernat** [comment]
I am using

```
opacity-rule = [
  "0:_NET_WM_STATE@[*]:32a = '_NET_WM_STATE_HIDDEN'",
  "100:_GTK_FRAME_EXTENTS@:c",
  "100:fullscreen",
  "100:name = 'Zoom Meeting'",
  "100:name = 'zoom_linux_float_video_window'",
  "100:class_g = 'Rofi'",
  "100:class_g = 'mpv'",
  "100:window_type = 'dock'",
  "100:window_type = 'utility'",
  "85:!focused"
];
```

It works for me. I don't have any other opacity directives.

**Reply by josianjehu** [reply]
I was actually able to fix my issue by adding `"_NET_WM_NAME@:s = 'rofi'"` to the focus-exception section in transparency section. 

I found my fix here (https://github.com/baskerville/bspwm/issues/755)
